examples/simulation_pytorch/README.md

```diff
@@ -1,69 +1,26 @@
-# Single-Machine Simulation of Federated Learning Systems
+# example_simulation_ray
```

Comment:
Shall we have a more descriptive title? E.g. `Single-Machine Simulation using simulation API` or similar?